doc/source/whatsnew/v1.3.2.rst

```diff
@@ -23,7 +23,7 @@ Fixed regressions
 
 Bug fixes
 ~~~~~~~~~
--
+- Fixed bug in :meth:`DataFrame.shift` where TypeError occurred when shifting DataFrame created by concatenation of slices and fills with values (:issue:`42719`)
```

Comment:
can you move to regression section above (may need to merge master first to avoid conflicts)

Reply:
Done.